td_core/db_work/days_7.py

```
@@ -11,18 +11,28 @@
 
 # sql_for_mdwiki.mdwiki_sql(query , update = False)
 # ---
-qua = '''
+qua = """
+select * from pages
+where target = '' and date < DATE_SUB(current_timestamp(), INTERVAL 7 DAY)
+"""
+# ---
+pages = sql_for_mdwiki.mdwiki_sql(qua)
+# ---
+for n, page in enumerate(pages, start=1):
```

Ruff

Loop control variable `n` not used within loop body

Rename unused `n` to `_n`

(B007)